_layouts/home.html

```
@@ -37,13 +39,13 @@
       {% endif %}
 
       <a href="{{ post.url | absolute_url }}">
-        <h2 class="post-title">{{ post.title | strip_html }}</h2>
+        <h3 style="font-size: 1.5rem; font-weight: 600;" class="post-title">{{ post.title | strip_html }}</h2>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </h2> ]

(tag-pair)